src/Multicall.sol

```diff
@@ -1,8 +1,5 @@
 // SPDX-License-Identifier: MIT
-
-
-pragma solidity >=0.5.0;
-pragma experimental ABIEncoderV2;
+pragma solidity 0.8.11;
```

Comment:
Since we're changing the pragmas in these files, we should add a note to the README that the source for the deployed versions of Multicall and Multicall2 can be found in commit bb309a985379c40bdbbc9a8613501732ed98bb9c or earlier